.changeset/chilly-ravens-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/eslint-plugin': minor
```

Comment:
Making a rule less strict, or removing a rule, can be a minor release (won't break linting on projects which already pass). Making a rule more strict, or adding a rule, should be a major release, since it may break linting on projects which update.